calculator.js

```diff
@@ -0,0 +1,119 @@
+(function() {
+
+    var inputField = document.getElementById('screen');
+    var currNumber = 0;
+    var result = 0;
+
+    var buttons = document.querySelector('.allButtons');
+    var numberButtons = buttons.querySelectorAll('.number');
+    /*=======================================================================*/
+
+    function Screen() {
+
+        this.element = inputField;
+    }
+
+    var screen = new Screen();
+
+    Screen.prototype.setNumber = function() {
+
+        this.element = (function() {
```

Comment:
WTF? You set element to input field and then to execution of an anonymous function which returns nothing (undefined)?